packages/gestalt/src/ButtonLink.js

```diff
@@ -0,0 +1,200 @@
+// @flow strict
+import { type AbstractComponent, forwardRef, type Node, useImperativeHandle, useRef } from 'react';
+import getAriaLabel from './accessibility/getAriaLabel.js';
+import NewTabAccessibilityLabel from './accessibility/NewTabAccessibilityLabel.js';
+import { useColorScheme } from './contexts/ColorSchemeProvider.js';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider.js';
+import Flex from './Flex.js';
+import Icon from './Icon.js';
+import icons from './icons/index.js';
+import InternalLink from './Link/InternalLink.js';
+import Text from './Text.js';
+
+const DEFAULT_TEXT_COLORS = {
+  blue: 'inverse',
+  gray: 'default',
+  red: 'inverse',
+  transparent: 'default',
+  semiTransparentWhite: 'default',
+  transparentWhiteText: 'inverse',
+  white: 'default',
+};
+
+const SIZE_NAME_TO_PIXEL = {
+  sm: 10,
+  md: 12,
+  lg: 12,
+};
+
+type Target = null | 'self' | 'blank';
+
+type ButtonType = {|
+  /**
+   * Label to provide more context around Button’s function or purpose. See the [Accessibility guidelines](/foundations/accessibility) to learn more.,
+   */
+  accessibilityLabel?: string,
+  /**
+   * The background color of Button.
+   */
+  color?:
+    | 'gray'
+    | 'red'
+    | 'blue'
+    | 'transparent'
+    | 'semiTransparentWhite'
+    | 'transparentWhiteText'
+    | 'white',
+  /**
+   * Available for testing purposes, if needed. Consider [better queries](https://testing-library.com/docs/queries/about/#priority) before using this prop.
+   */
+  dataTestId?: string,
+  /**
+   * Indicates if Button is disabled. Disabled Buttons are inactive and cannot be interacted with.
+   */
+  disabled?: boolean,
+  /**
+   * An icon displayed after the text to help clarify the usage of Button. See the [icon variant](#Icons) to learn more.
+   */
+  iconEnd?: $Keys<typeof icons>,
+  /**
+   * Default Buttons are sized by the text within the Button whereas full-width Buttons expand to the full width of their container.
+   */
+  fullWidth?: boolean,
+  /**
+   * Default Buttons are sized by the text within the Button whereas full-width Buttons expand to the full width of their container.
+   */
+  tabIndex?: -1 | 0,
+  /**
+   * Callback invoked when the user clicks (press and release) on Button with the mouse or keyboard. Required with `role="button"` or `type="button"` Buttons.
+     See [GlobalEventsHandlerProvider](/web/utilities/globaleventshandlerprovider#Link-handlers) to learn more about link navigation.
+   */
+  onClick?: ({|
+    event:
+      | SyntheticMouseEvent<HTMLButtonElement>
```

Comment:
this is not a button anymore, just anchors

Reply:
DONE